RecoHI/Configuration/python/Reconstruction_hiPF_cff.py

```diff
@@ -16,6 +16,7 @@
 gedPhotonsTmp.isolationSumsCalculatorSet.trackProducer = cms.InputTag("hiGeneralTracks")
 gedPhotons.primaryVertexProducer = cms.InputTag("hiSelectedVertex")
 gedPhotons.isolationSumsCalculatorSet.trackProducer = cms.InputTag("hiGeneralTracks")
+photonIDValueMapProducerRECO.vertices = cms.InputTag("hiSelectedVertex")
```

Comment:
this needs a fix

Reply:
urgh right  you are, sorry, taking shortcuts always ends up bitting me :(